[bug] tauri-cli v2.9.0 does not compile

### Describe the bug

The command `cargo install tauri-cli` fails with:
```
error[E0412]: cannot find type `Error` in crate `tauri_macos_sign`
   --> /Users/ctm/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/tauri-bundler-2.7.0/src/error.rs:171:47
    |
171 |   AppleCodesign(#[from] Box<tauri_macos_sign::Error>),
    |                                               ^^^^^ not found in `tauri_macos_sign`
    |
help: consider importing one of these items
    |
  6 + use crate::Error;
    |
  6 + use crate::error::io::Error;
    |
  6 + use std::error::Error;
    |
  6 + use std::fmt::Error;
    |
    = and 22 other candidates
help: if you import `Error`, refer to it directly
    |
171 -   AppleCodesign(#[from] Box<tauri_macos_sign::Error>),
171 +   AppleCodesign(#[from] Box<Error>),
    |
```

### Reproduction

My guess is it compiles fine for most other people, since I appear to be the first one reporting this, but on my mac laptop running 15.7.1 (24G231), `cargo install tauri-cli` produces the following output:
```
cargo install tauri-cli
    Updating crates.io index
  Installing tauri-cli v2.9.0
    Updating crates.io index
     Locking 863 packages to latest compatible versions
      Adding cargo-mobile2 v0.21.2 (available: v0.22.1)
      Adding colored v2.2.0 (available: v3.0.0)
      Adding dialoguer v0.11.0 (available: v0.12.0)
      Adding elf v0.7.4 (available: v0.8.0)
      Adding html5ever v0.29.1 (available: v0.35.0)
      Adding itertools v0.13.0 (available: v0.14.0)
      Adding json-patch v3.0.1 (available: v4.1.0)
      Adding jsonrpsee v0.24.9 (available: v0.26.0)
      Adding jsonrpsee-client-transport v0.24.9 (available: v0.26.0)
      Adding jsonrpsee-core v0.24.9 (available: v0.26.0)
      Adding jsonrpsee-ws-client v0.24.9 (available: v0.26.0)
      Adding matchit v0.8.4 (available: v0.8.6)
      Adding minisign v0.7.3 (available: v0.7.9)
      Adding object v0.36.7 (available: v0.37.3)
      Adding oxc_allocator v0.36.0 (available: v0.95.0)
      Adding oxc_ast v0.36.0 (available: v0.95.0)
      Adding oxc_parser v0.36.0 (available: v0.95.0)
      Adding oxc_span v0.36.0 (available: v0.95.0)
      Adding phf v0.11.3 (available: v0.13.1)
      Adding windows-sys v0.60.2 (available: v0.61.2)
      Adding zip v4.6.1 (available: v6.0.0)
   Compiling proc-macro2 v1.0.101
   Compiling quote v1.0.41
   Compiling unicode-ident v1.0.19
   Compiling libc v0.2.177
   Compiling cfg-if v1.0.4
   Compiling version_check v0.9.5
   Compiling serde_core v1.0.228
   Compiling memchr v2.7.6
   Compiling value-bag v1.11.1
   Compiling serde v1.0.228
   Compiling itoa v1.0.15
   Compiling typenum v1.19.0
   Compiling smallvec v1.15.1
   Compiling subtle v2.6.1
   Compiling once_cell v1.21.3
   Compiling bytes v1.10.1
   Compiling hashbrown v0.16.0
   Compiling generic-array v0.14.9
   Compiling libm v0.2.15
   Compiling equivalent v1.0.2
   Compiling bitflags v2.10.0
   Compiling log v0.4.28
   Compiling autocfg v1.5.0
   Compiling const-oid v0.9.6
   Compiling indexmap v2.12.0
   Compiling syn v2.0.107
   Compiling pin-project-lite v0.2.16
   Compiling num-traits v0.2.19
   Compiling fnv v1.0.7
   Compiling futures-core v0.3.31
   Compiling shlex v1.3.0
   Compiling zerocopy v0.8.27
   Compiling find-msvc-tools v0.1.4
   Compiling thiserror v1.0.69
   Compiling futures-sink v0.3.31
   Compiling cc v1.2.41
   Compiling ryu v1.0.20
   Compiling futures-io v0.3.31
   Compiling getrandom v0.2.16
   Compiling slab v0.4.11
   Compiling stable_deref_trait v1.2.1
   Compiling futures-channel v0.3.31
   Compiling pin-utils v0.1.0
   Compiling rand_core v0.6.4
   Compiling futures-task v0.3.31
   Compiling mio v1.1.0
   Compiling socket2 v0.6.1
   Compiling getrandom v0.3.4
   Compiling crossbeam-utils v0.8.21
   Compiling serde_json v1.0.145
   Compiling core-foundation-sys v0.8.7
   Compiling http v1.3.1
   Compiling tracing-core v0.1.34
   Compiling simd-adler32 v0.3.7
   Compiling httparse v1.10.1
   Compiling cpufeatures v0.2.17
   Compiling base64 v0.22.1
   Compiling percent-encoding v2.3.2
   Compiling ring v0.17.14
   Compiling syn v1.0.109
   Compiling untrusted v0.9.0
   Compiling adler2 v2.0.1
   Compiling miniz_oxide v0.8.9
   Compiling writeable v0.6.1
   Compiling litemap v0.8.0
   Compiling icu_properties_data v2.0.1
   Compiling crc32fast v1.5.0
   Compiling icu_normalizer_data v2.0.0
   Compiling parking_lot_core v0.9.12
   Compiling rustix v1.1.2
   Compiling ppv-lite86 v0.2.21
   Compiling errno v0.3.14
   Compiling scopeguard v1.2.0
   Compiling zlib-rs v0.5.2
   Compiling byteorder v1.5.0
   Compiling tower-service v0.3.3
   Compiling base64ct v1.8.0
   Compiling lock_api v0.4.14
   Compiling rand_chacha v0.3.1
   Compiling pem-rfc7468 v0.7.0
   Compiling rand v0.8.5
   Compiling form_urlencoded v1.2.2
   Compiling num-integer v0.1.46
   Compiling aho-corasick v1.1.3
   Compiling regex-syntax v0.8.8
   Compiling try-lock v0.2.5
   Compiling utf8_iter v1.0.4
   Compiling siphasher v1.0.1
   Compiling synstructure v0.13.2
   Compiling phf_shared v0.11.3
   Compiling want v0.3.1
   Compiling rayon-core v1.13.0
   Compiling httpdate v1.0.3
   Compiling phf_generator v0.11.3
   Compiling ident_case v1.0.1
   Compiling bitflags v1.3.2
   Compiling strsim v0.11.1
   Compiling num-bigint v0.4.6
   Compiling security-framework-sys v2.15.0
   Compiling arrayvec v0.7.6
   Compiling unicode-width v0.2.2
   Compiling getrandom v0.1.16
   Compiling siphasher v0.3.11
   Compiling crossbeam-epoch v0.9.18
   Compiling parking_lot v0.12.5
   Compiling http-body v1.0.1
   Compiling iana-time-zone v0.1.64
   Compiling ahash v0.8.12
   Compiling pkg-config v0.3.32
   Compiling ipnet v2.11.0
   Compiling zeroize_derive v1.4.2
   Compiling serde_derive v1.0.228
   Compiling zerocopy-derive v0.8.27
   Compiling thiserror-impl v1.0.69
   Compiling futures-macro v0.3.31
   Compiling zeroize v1.8.2
   Compiling zerofrom-derive v0.1.6
   Compiling futures-util v0.3.31
   Compiling tokio-macros v2.6.0
   Compiling crypto-common v0.1.6
   Compiling block-buffer v0.10.4
   Compiling yoke-derive v0.8.0
   Compiling digest v0.10.7
   Compiling displaydoc v0.2.5
   Compiling tokio v1.48.0
   Compiling zerovec-derive v0.11.1
   Compiling zerofrom v0.1.6
   Compiling tracing-attributes v0.1.30
   Compiling yoke v0.8.0
   Compiling zerotrie v0.2.2
   Compiling libz-rs-sys v0.5.2
   Compiling flate2 v1.1.4
   Compiling zerovec v0.11.4
   Compiling der v0.7.10
   Compiling regex-automata v0.4.13
   Compiling tracing v0.1.41
   Compiling block-padding v0.3.3
   Compiling inout v0.1.4
   Compiling sha1 v0.10.6
   Compiling cipher v0.4.4
   Compiling rustls-pki-types v1.12.0
   Compiling utf-8 v0.7.6
   Compiling tinystr v0.8.1
   Compiling potential_utf v0.1.3
   Compiling spki v0.7.3
   Compiling icu_locale_core v2.0.0
   Compiling icu_collections v2.0.0
   Compiling minimal-lexical v0.2.1
   Compiling chrono v0.4.42
   Compiling nom v7.1.3
   Compiling crossbeam-deque v0.8.6
   Compiling icu_provider v2.0.0
   Compiling core-foundation v0.9.4
   Compiling icu_normalizer v2.0.0
   Compiling icu_properties v2.0.1
   Compiling spin v0.9.8
   Compiling static_assertions v1.1.0
   Compiling lazy_static v1.5.0
   Compiling rand_core v0.5.1
   Compiling futures-executor v0.3.31
   Compiling http v0.2.12
   Compiling rustls v0.21.12
   Compiling tokio-util v0.7.16
   Compiling idna_adapter v1.2.1
   Compiling idna v1.1.0
   Compiling winnow v0.7.13
   Compiling either v1.15.0
   Compiling url v2.5.7
   Compiling regex v1.12.2
   Compiling new_debug_unreachable v1.0.6
   Compiling anyhow v1.0.100
   Compiling futures v0.3.31
   Compiling security-framework v2.11.1
   Compiling serde_urlencoded v0.7.1
   Compiling rustls-webpki v0.101.7
   Compiling sct v0.7.1
   Compiling signature v2.2.0
   Compiling phf_macros v0.11.3
   Compiling same-file v1.0.6
   Compiling rustc-hash v2.1.1
   Compiling bytemuck v1.24.0
   Compiling sha1_smol v1.0.1
   Compiling system-configuration-sys v0.5.0
   Compiling thiserror v2.0.17
   Compiling allocator-api2 v0.2.21
   Compiling uuid v1.18.1
   Compiling phf v0.11.3
   Compiling walkdir v2.5.0
   Compiling bumpalo v3.19.0
   Compiling h2 v0.3.27
   Compiling http-body v0.4.6
   Compiling rand_chacha v0.2.2
   Compiling rand_pcg v0.2.1
   Compiling hmac v0.12.1
   Compiling thiserror-impl v2.0.17
   Compiling http-body-util v0.1.3
   Compiling terminal_size v0.4.3
   Compiling pem v3.0.6
   Compiling socket2 v0.5.10
   Compiling bcder v0.7.6
   Compiling encoding_rs v0.8.35
   Compiling base64 v0.21.7
   Compiling tower-layer v0.3.3
   Compiling owo-colors v4.2.3
   Compiling unicode-linebreak v0.1.5
   Compiling utf8parse v0.2.2
   Compiling hex v0.4.3
   Compiling mime v0.3.17
   Compiling powerfmt v0.2.0
   Compiling convert_case v0.4.0
   Compiling proc-macro-hack v0.5.20+deprecated
   Compiling ucd-trie v0.1.7
   Compiling smawk v0.3.2
   Compiling rustversion v1.0.22
   Compiling textwrap v0.16.2
   Compiling derive_more v0.99.20
   Compiling tokio-rustls v0.24.1
   Compiling deranged v0.5.4
   Compiling anstyle-parse v0.2.7
   Compiling rustls-pemfile v1.0.4
   Compiling rand v0.7.3
   Compiling pest v2.8.3
   Compiling hashbrown v0.14.5
   Compiling pkcs8 v0.10.2
   Compiling sha2 v0.10.9
   Compiling num-rational v0.4.2
   Compiling bzip2-sys v0.1.13+1.0.8
   Compiling lzma-sys v0.1.20
   Compiling equator-macro v0.4.2
   Compiling scroll_derive v0.12.1
   Compiling phf_shared v0.8.0
   Compiling phf_shared v0.10.0
   Compiling darling_core v0.20.11
   Compiling num-iter v0.1.45
   Compiling proc-macro2-diagnostics v0.10.1
   Compiling atomic-waker v1.1.2
   Compiling anstyle-query v1.1.4
   Compiling anstyle v1.0.13
   Compiling typewit_proc_macros v1.8.1
   Compiling object v0.37.3
   Compiling hyper v0.14.32
   Compiling semver v1.0.27
   Compiling time-core v0.1.6
   Compiling radium v0.7.0
   Compiling num-conv v0.1.0
   Compiling is_terminal_polyfill v1.70.1
   Compiling colorchoice v1.0.4
   Compiling time v0.3.44
   Compiling anstream v0.6.21
   Compiling rustc_version v0.4.1
   Compiling typewit v1.14.2
   Compiling scroll v0.12.0
   Compiling h2 v0.4.12
   Compiling equator v0.4.2
   Compiling phf_generator v0.10.0
   Compiling phf_generator v0.8.0
   Compiling pest_meta v2.8.3
   Compiling system-configuration v0.5.1
   Compiling rustls-native-certs v0.6.3
   Compiling rayon v1.11.0
   Compiling darling_macro v0.20.11
   Compiling oxc-miette-derive v1.0.2
   Compiling pin-project-internal v1.1.10
   Compiling string_cache_codegen v0.5.4
   Compiling phf_codegen v0.11.3
   Compiling fdeflate v0.3.7
   Compiling rand_core v0.9.3
   Compiling ff v0.13.1
   Compiling quick-xml v0.38.3
   Compiling hyper-rustls v0.24.2
   Compiling yansi v1.0.1
   Compiling webpki-roots v0.25.4
   Compiling sync_wrapper v0.1.2
   Compiling unicode-xid v0.2.6
   Compiling precomputed-hash v0.1.1
   Compiling gimli v0.32.3
   Compiling doc-comment v0.3.3
   Compiling option-ext v0.2.0
   Compiling base16ct v0.2.0
   Compiling tap v1.0.1
   Compiling data-encoding v2.9.0
   Compiling mac v0.1.1
   Compiling wyz v0.5.1
   Compiling futf v0.1.5
   Compiling sec1 v0.7.3
   Compiling plist v1.8.0
   Compiling synstructure v0.12.6
   Compiling hyper v1.7.0
   Compiling reqwest v0.11.27
   Compiling oxc-miette v1.0.2
   Compiling pin-project v1.1.10
   Compiling group v0.13.0
   Compiling rand_chacha v0.9.0
   Compiling addr2line v0.25.1
   Compiling markup5ever v0.14.1
   Compiling phf_macros v0.10.0
   Compiling darling v0.20.11
   Compiling castaway v0.2.4
   Compiling pest_generator v2.8.3
   Compiling phf_codegen v0.8.0
   Compiling aligned-vec v0.6.4
   Compiling curve25519-dalek v4.1.3
   Compiling x509-certificate v0.23.1
   Compiling rustls-pemfile v2.2.0
   Compiling serde_spanned v0.6.9
   Compiling toml_datetime v0.6.11
   Compiling cssparser v0.29.6
   Compiling crypto-bigint v0.5.5
   Compiling darling_core v0.21.3
   Compiling os_pipe v1.2.3
   Compiling sync_wrapper v1.0.2
   Compiling uncased v0.9.10
   Compiling funty v2.0.0
   Compiling toml_write v0.1.2
   Compiling signal-hook v0.3.18
   Compiling rustc-demangle v0.1.26
   Compiling clap_lex v0.7.6
   Compiling dtoa v1.0.10
   Compiling quick-error v2.0.1
   Compiling built v0.7.7
   Compiling rustls v0.22.4
   Compiling simple-file-manifest v0.11.0
   Compiling xml-rs v0.8.27
   Compiling heck v0.5.0
   Compiling paste v1.0.15
   Compiling heck v0.4.1
   Compiling num-bigint-dig v0.8.4
   Compiling typeid v1.0.3
   Compiling snafu-derive v0.7.5
   Compiling clap_derive v4.5.49
   Compiling rav1e v0.7.1
   Compiling dtoa-short v0.3.5
   Compiling clap_builder v4.5.50
   Compiling elliptic-curve v0.13.8
   Compiling darling_macro v0.21.3
   Compiling serde-xml-rs v0.6.0
   Compiling toml_edit v0.22.27
   Compiling bitvec v1.0.1
   Compiling cryptographic-message-syntax v0.26.0
   Compiling pest_derive v2.8.3
   Compiling backtrace v0.3.76
   Compiling xz2 v0.1.7
   Compiling phf v0.10.1
   Compiling v_frame v0.3.9
   Compiling selectors v0.24.0
   Compiling bzip2 v0.4.4
   Compiling compact_str v0.8.1
   Compiling derive_builder_core v0.20.2
   Compiling hyper-util v0.1.17
   Compiling rand v0.9.2
   Compiling asn1-rs-derive v0.4.0
   Compiling pear_codegen v0.2.9
   Compiling tendril v0.4.3
   Compiling itertools v0.10.5
   Compiling string_cache v0.8.9
   Compiling konst_kernel v0.3.15
   Compiling const_panic v0.2.15
   Compiling oxc_allocator v0.36.0
   Compiling toml_parser v1.0.4
   Compiling tokio-stream v0.1.17
   Compiling rusticata-macros v4.1.0
   Compiling twox-hash v1.6.3
   Compiling rustls-webpki v0.102.8
   Compiling jsonrpsee-types v0.24.9
   Compiling asn1-rs-impl v0.1.0
   Compiling md-5 v0.10.6
   Compiling universal-hash v0.5.1
   Compiling async-trait v0.1.89
   Compiling oxc_ast_macros v0.36.0
   Compiling profiling-procmacros v1.0.17
   Compiling curve25519-dalek-derive v0.1.1
   Compiling cssparser-macros v0.6.1
   Compiling toml_datetime v0.7.3
   Compiling serde_spanned v1.0.3
   Compiling core-foundation v0.10.1
   Compiling core_maths v0.1.1
   Compiling signal-hook-registry v1.4.6
   Compiling slotmap v1.0.7
   Compiling figment v0.10.19
   Compiling unic-common v0.9.0
   Compiling float-cmp v0.9.0
   Compiling oid-registry v0.6.1
   Compiling weezl v0.1.10
   Compiling ref-cast v1.0.25
   Compiling rustls v0.23.33
   Compiling is_executable v1.0.5
   Compiling plain v0.2.3
   Compiling futures-timer v3.0.3
   Compiling opaque-debug v0.3.1
   Compiling oxc_estree v0.36.0
   Compiling zune-core v0.4.12
   Compiling fastrand v2.3.0
   Compiling unic-char-range v0.9.0
   Compiling erased-serde v0.4.8
   Compiling nodrop v0.1.14
   Compiling toml_writer v1.0.4
   Compiling minicbor v0.20.0
   Compiling matches v0.1.10
   Compiling inlinable_string v0.1.15
   Compiling tempfile v3.23.0
   Compiling zune-jpeg v0.4.21
   Compiling servo_arc v0.2.0
   Compiling pear v0.2.9
   Compiling jsonrpsee-core v0.24.9
   Compiling unic-char-property v0.9.0
   Compiling oxc_span v0.36.0
   Compiling polyval v0.6.2
   Compiling cpio-archive v0.9.0
   Compiling strict-num v0.1.1
   Compiling unic-ucd-version v0.9.0
   Compiling ttf-parser v0.25.1
   Compiling profiling v1.0.17
   Compiling asn1-rs v0.5.2
   Compiling apple-xar v0.18.0
   Compiling rasn-derive v0.12.5
   Compiling ruzstd v0.5.0
   Compiling konst v0.3.16
   Compiling phf v0.8.0
   Compiling snafu v0.7.5
   Compiling bitvec-nom2 v0.2.1
   Compiling derive_builder_macro v0.20.2
   Compiling clap v4.5.50
   Compiling toml v0.8.23
   Compiling av1-grain v0.2.4
   Compiling darling v0.21.3
   Compiling primeorder v0.13.6
   Compiling tower v0.5.2
   Compiling rustls-native-certs v0.7.3
   Compiling maybe-rayon v0.1.1
   Compiling dirs-sys v0.4.1
   Compiling png v0.17.16
   Compiling pkcs1 v0.7.5
   Compiling hkdf v0.12.4
   Compiling rgb v0.8.52
   Compiling cookie-factory v0.3.3
   Compiling soketto v0.8.1
   Compiling itertools v0.12.1
   Compiling rustls-webpki v0.103.7
   Compiling cbc v0.1.2
   Compiling rc2 v0.8.1
   Compiling des v0.8.1
   Compiling half v2.7.1
   Compiling minicbor-derive v0.13.0
   Compiling fax_derive v0.2.0
   Compiling ref-cast-impl v1.0.25
   Compiling arg_enum_proc_macro v0.3.4
   Compiling num-derive v0.4.2
   Compiling match_token v0.1.0
   Compiling console v0.15.11
   Compiling serde_derive_internals v0.29.1
   Compiling fxhash v0.2.1
   Compiling filetime v0.2.26
   Compiling is-terminal v0.4.16
   Compiling simd_helpers v0.1.0
   Compiling cfg_aliases v0.2.1
   Compiling yasna v0.5.2
   Compiling color_quant v1.1.0
   Compiling bitstream-io v2.6.0
   Compiling humantime v2.3.0
   Compiling unicode-id-start v1.4.0
   Compiling shell-words v1.1.0
   Compiling imgref v1.12.0
   Compiling byteorder-lite v0.1.0
   Compiling arrayref v0.3.9
   Compiling schemars v0.8.22
   Compiling unsafe-libyaml v0.2.11
   Compiling glob v0.3.3
   Compiling noop_proc_macro v0.3.0
   Compiling num-modular v0.6.1
   Compiling jzon v0.12.5
   Compiling bytesize v1.3.3
   Compiling tinyvec_macros v0.1.1
   Compiling termcolor v1.4.1
   Compiling zip_structs v0.2.1
   Compiling tinyvec v1.10.0
   Compiling rasn v0.12.5
   Compiling num-order v1.2.0
   Compiling serde_yaml v0.9.34+deprecated
   Compiling env_logger v0.10.2
   Compiling image-webp v0.2.4
   Compiling schemars_derive v0.8.22
   Compiling tiny-skia-path v0.11.4
   Compiling p12 v0.6.3
   Compiling loop9 v0.1.5
   Compiling dialoguer v0.11.0
   Compiling gif v0.13.3
   Compiling nix v0.30.1
   Compiling html5ever v0.29.1
   Compiling fax v0.2.6
   Compiling x509 v0.2.0
   Compiling rsa v0.9.8
   Compiling spake2 v0.4.0
   Compiling dirs v5.0.1
   Compiling tungstenite v0.21.0
   Compiling p256 v0.13.2
   Compiling serde_with_macros v3.15.0
   Compiling derive_builder v0.20.2
   Compiling apple-flat-package v0.18.0
   Compiling object v0.32.2
   Compiling sigchld v0.2.4
   Compiling unic-ucd-ident v0.9.0
   Compiling ghash v0.5.1
   Compiling toml v0.9.8
   Compiling goblin v0.8.2
   Compiling security-framework v3.5.1
   Compiling apple-bundles v0.19.0
   Compiling oxc_diagnostics v0.36.0
   Compiling dirs-sys v0.5.0
   Compiling dashmap v6.1.0
   Compiling cfb v0.7.3
   Compiling once-cell-regex v0.2.1
   Compiling ctr v0.9.2
   Compiling aes v0.8.4
   Compiling jsonptr v0.6.3
   Compiling zip v0.6.6
   Compiling aead v0.5.2
   Compiling serialize-to-javascript-impl v0.1.2
   Compiling kurbo v0.11.3
   Compiling avif-serialize v0.8.6
   Compiling pxfm v0.1.25
   Compiling num-complex v0.4.6
   Compiling zune-inflate v0.2.54
   Compiling memmap2 v0.9.8
   Compiling bstr v1.12.0
   Compiling dunce v1.0.5
   Compiling difference v2.0.0
   Compiling unicode-properties v0.1.3
   Compiling unicode-script v0.5.7
   Compiling unicode-ccc v0.4.0
   Compiling iri-string v0.7.8
   Compiling assert-unchecked v0.1.2
   Compiling nonmax v0.5.5
   Compiling borrow-or-share v0.2.2
   Compiling bit-vec v0.8.0
   Compiling lebe v0.5.3
   Compiling bit_field v0.10.3
   Compiling env_home v0.1.0
   Compiling oxc_index v0.36.0
   Compiling dyn-clone v1.0.20
   Compiling unicode-bidi-mirroring v0.4.0
   Compiling which v8.0.0
   Compiling rustybuzz v0.20.1
   Compiling exr v1.73.0
   Compiling oxc_syntax v0.36.0
   Compiling tower-http v0.6.6
   Compiling moxcms v0.7.7
   Compiling bit-set v0.8.0
   Compiling fluent-uri v0.3.2
   Compiling globset v0.4.17
   Compiling apple-codesign v0.27.0
   Compiling svgtypes v0.15.3
   Compiling fontdb v0.23.0
   Compiling num v0.4.3
   Compiling ravif v0.11.20
   Compiling serialize-to-javascript v0.1.2
   Compiling aes-gcm v0.10.3
   Compiling json-patch v3.0.1
   Compiling infer v0.19.0
   Compiling dirs v6.0.0
   Compiling oxc_regular_expression v0.36.0
   Compiling urlpattern v0.3.0
   Compiling shared_child v1.1.1
   Compiling serde_with v3.15.0
   Compiling handlebars v6.3.2
   Compiling serde-untagged v0.1.9
   Compiling tiff v0.10.3
   Compiling kuchikiki v0.8.8-speedreader
   Compiling tungstenite v0.28.0
   Compiling json5 v0.4.1
   Compiling tower v0.4.13
   Compiling png v0.18.0
   Compiling pbkdf2 v0.12.2
   Compiling zopfli v0.8.2
   Compiling qoi v0.4.1
   Compiling webpki-roots v1.0.3
   Compiling salsa20 v0.10.2
   Compiling ctor v0.2.9
   Compiling xattr v1.6.1
   Compiling socks v0.3.4
   Compiling ureq-proto v0.5.2
   Compiling fsevent-sys v4.1.0
   Compiling rtoolbox v0.0.3
   Compiling simplecss v0.2.2
   Compiling unicode-bidi v0.3.18
   Compiling xmlwriter v0.1.0
   Compiling object v0.36.7
   Compiling shared_thread v0.2.0
   Compiling cargo-mobile2 v0.21.2
   Compiling roxmltree v0.20.0
   Compiling unicode-vo v0.1.0
   Compiling route-recognizer v0.3.1
   Compiling vsimd v0.8.0
   Compiling notify-types v2.0.0
   Compiling std_prelude v0.2.12
   Compiling cow-utils v0.1.3
   Compiling imagesize v0.13.0
   Compiling data-url v0.3.2
   Compiling pico-args v0.5.0
   Compiling outref v0.5.2
   Compiling stfu8 v0.2.7
   Compiling path_abs v0.5.1
   Compiling usvg v0.45.1
   Compiling uuid-simd v0.8.0
   Compiling oxc_ast v0.36.0
   Compiling notify v8.2.0
   Compiling jsonrpsee-server v0.24.9
   Compiling duct v1.1.0
   Compiling rpassword v7.4.0
   Compiling tauri-utils v2.8.0
   Compiling rustls-platform-verifier v0.6.1
   Compiling tar v0.4.44
   Compiling ureq v3.1.2
   Compiling image v0.25.8
   Compiling zip v4.6.1
   Compiling scrypt v0.11.0
   Compiling tauri-macos-sign v2.2.0
   Compiling tokio-tungstenite v0.28.0
   Compiling fraction v0.15.3
   Compiling referencing v0.33.0
   Compiling ignore v0.4.24
   Compiling fancy-regex v0.16.2
   Compiling reqwest v0.12.24
   Compiling tiny-skia v0.11.4
   Compiling jsonrpsee-client-transport v0.24.9
   Compiling tauri-icns v0.1.0
   Compiling goblin v0.9.3
   Compiling axum-core v0.5.5
   Compiling os_info v3.12.0
   Compiling java-properties v2.0.0
   Compiling x509-certificate v0.24.0
   Compiling env_filter v0.1.4
   Compiling colored v2.2.0
   Compiling email_address v0.2.9
   Compiling serde_path_to_error v0.1.20
   Compiling ordered-float v2.10.1
   Compiling include_dir_macros v0.7.4
   Compiling tauri-cli v2.9.0
   Compiling home v0.5.11
   Compiling file-id v0.2.3
   Compiling vlq v0.5.1
   Compiling dispatch v0.2.0
   Compiling deunicode v1.6.2
   Compiling num-cmp v0.1.0
   Compiling jiff v0.2.15
   Compiling bytecount v0.6.9
   Compiling seq-macro v0.3.6
   Compiling matchit v0.8.4
   Compiling english-numbers v0.3.3
   Compiling base64 v0.13.1
   Compiling magic_string v0.3.4
   Compiling oxc_parser v0.36.0
   Compiling axum v0.8.6
   Compiling env_logger v0.11.8
   Compiling jsonschema v0.33.0
   Compiling ctrlc v3.5.0
   Compiling notify-debouncer-full v0.6.0
   Compiling include_dir v0.7.4
   Compiling serde-value v0.7.0
   Compiling tauri-bundler v2.7.0
error[E0412]: cannot find type `Error` in crate `tauri_macos_sign`
   --> /Users/ctm/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/tauri-bundler-2.7.0/src/error.rs:171:47
    |
171 |   AppleCodesign(#[from] Box<tauri_macos_sign::Error>),
    |                                               ^^^^^ not found in `tauri_macos_sign`
    |
help: consider importing one of these items
    |
  6 + use crate::Error;
    |
  6 + use crate::error::io::Error;
    |
  6 + use std::error::Error;
    |
  6 + use std::fmt::Error;
    |
    = and 22 other candidates
help: if you import `Error`, refer to it directly
    |
171 -   AppleCodesign(#[from] Box<tauri_macos_sign::Error>),
171 +   AppleCodesign(#[from] Box<Error>),
    |

   Compiling resvg v0.45.1
For more information about this error, try `rustc --explain E0412`.
error: could not compile `tauri-bundler` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `tauri-cli v2.9.0`, intermediate artifacts can be found at `/var/folders/mx/hdmj2f6n7pb06t986m403jmm0000gn/T/cargo-install8LrSBX`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```
I'm using the latest stable rust:
```
rustc --version
rustc 1.90.0 (1159e78c4 2025-09-14)
```

### Expected behavior

I expect tauri-cli to install. I've been using tauri-cli since late July, upgrading it each time a new version has been available and this is the first time it has failed to compile.

### Full `tauri info` output

```text
% cargo tauri info
error: no such command: `tauri`

help: a command with a similar name exists: `audit`

help: view all installed commands with `cargo --list`
help: find a package to install `tauri` with `cargo search cargo-tauri`
```

### Stack trace

```text

```

### Additional context

_No response_

--- a/.changes/macos-codesign-error-enum.md
+++ b/.changes/macos-codesign-error-enum.md
@@ -1,5 +1,5 @@
 ---
-tauri-macos-codesign: "minor:enhance"
+tauri-macos-sign: "minor:enhance"
 ---
 
 **Potentially breaking change:** Export custom Error enum instead of using anyhow. The changes happened in https://github.com/tauri-apps/tauri/pull/14126.

--- a/.changes/macos-codesign-error-enum.md
+++ b/.changes/macos-codesign-error-enum.md
@@ -1,0 +1,5 @@
+---
+tauri-macos-codesign: "minor:enhance"
+---
+
+**Potentially breaking change:** Export custom Error enum instead of using anyhow. The changes happened in https://github.com/tauri-apps/tauri/pull/14126.

